docs/cloud/pouta/persistent-volumes.md

```diff
@@ -22,6 +22,16 @@ is the size of the volume.
 
 ![Create persistent Volmume](../../img/create-volume-horizon.png)
 
+!!! Warning "No äkköset in name or description allowed"
+    This is a know bug in the volume system. Volume creation will fail
+    if its name or description contains any non ASCII characters.
+
+
+    ![Creating](../img/Creating.png) ![Unable to create volume](../img/Unable-to-create-volume.png)
```

Comment:
The image about the error is in line with what is written in the text, i.e., volume creation will fail. However, if you want to add also the image of the creation bar getting stuck, then I think it should be explained, e.g., by adding a sentence such as `After returning the error, the dashboard will show the volume as stuck in "creating" stage` or something like that

Reply:
Fixed